packages/legend-query-builder/src/components/__tests__/QueryBuilderParametersPanel.test.tsx

```diff
@@ -45,7 +45,7 @@ import {
 
 test(
   integrationTest(
-    'Query builder shows validation error for constant name if existing duplicate parameter name',
+    'Query builder shows validation error for parameter name if existing duplicate constant name',
```

Comment:
why do we change this? Don't we have validations on constant name if there is an existing paramter name anymore?